.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Changelist
+[Describe or list the changes made in this PR]
+
+### Test Plan
+[Describe how this PR was tested (if applicable)]
+
+### Author/Reviewer Checklist
+- Add an entry to `protocol/CHANGELOG.md` and/or `indexer/CHANGELOG.md` (if applicable).
```

Comment:
nit: can we make these into checkboxes so authors can check them off?

separately:
1. I don't see `CHANGELOG.md` - can you add them as part of this PR?
2. do you have guidelines around what to enter in the changelog? it would be good to add some examples so people can follow.